src/controller/products.controller.ts

```diff
@@ -32,7 +32,10 @@ export default function (
 
     fastify.get(
       '/',
-      async (req, res): Promise<{ featuresPerService: { antivirus: boolean; backups: boolean } } | Error> => {
+      async (
+        req,
+        res,
+      ): Promise<{ featuresPerService: { antivirus: boolean; backups: boolean; cleaner: boolean } } | Error> => {
```

Comment:
I believe this change is no longer necessary, as we will be using the other endpoint. It was temporary. You can ignore it.

Also, note that you must return `cleaner: <true | false>` in the `antivirusTier` object.